Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN npm i -g npm@8.10.0
 RUN npm i -g firebase-tools@11.16.0
 
 COPY LICENSE README.md /
+COPY .env* /
```

Comment:
I've an issue with functions `.env` files not being imported at build time. I'm trying to set region from `.env` which works for CLI deploy but not in a Github Workflow. So this would be helpful alright.

Worth noting that docs seem to indicate (in a roundabout way) that `.env.*` for functions should live in the `/functions` directory.
https://firebase.google.com/docs/functions/config-env#params